doc/manual/src/language/string-interpolation.md

```diff
@@ -0,0 +1,97 @@
+# String interpolation
+
+String interpolation is a language feature where a [string], [path], or [attribute name] can contain expressions enclosed in `${ }` (dollar-sign with curly brackets).
+
+Such a string is an *interpolated string*, and an expression inside is an *interpolated expression*.
+
+Interpolated expressions must evaluate to something that can be coerced to a string:
+
+- [string]
+- [path]
+- [derivation]
+
+[string]: ./language/values.md#type-string
+[path]: ./language/values.md#type-path
+[attribute name]: ./language/values.md#attribute-set
+[derivation]: ../glossary.md#gloss-derivation
+
+When an interpolated expression evaluates to a path, and is thus coerced into a string, the path is first copied into the Nix store and the resulting string is the [store path].
```

Comment:
"thus" is causal, but I think you want temporal relation here: "and is *then* coerced"

Reply:
Actually, the coercion part is irrelevant for that statement anyway.

```suggestion
When an interpolated expression evaluates to a path, the path is first copied into the Nix store and the resulting string is the [store path].
```